packages/firestore/src/protos/google/firestore/v1/query.proto

```diff
@@ -309,7 +309,7 @@ message StructuredAggregationQuery {
       // Requires:
       //
       // * Must be greater than zero when present.
-      int32 up_to = 1;
+      int64 up_to = 1;
```

Comment:
This has to be `google.protobuf.Int64Value`.

See https://github.com/firebase/firebase-android-sdk/pull/4038

Reply:
do we have google.protobuf.Int64Value in web sdk? 
web sdk used google.protobuf.Int32Value, but didn't see any line uses google.protobuf.Int64Value. 🤔

Reply:
Yes, I believe we do have `google.protobuf.Int64Value` in the web sdk. Even if we don't it's fine because we never use it. But it's important that the proto definition is correct because if we ever do use it in the future then we want it to work correctly. Are you encountering errors when you change it to Int64Value?

Reply:
nope, no errors